sql/migrations/0001_initial.sql

```diff
@@ -17,10 +17,23 @@ CREATE INDEX ballots_by_layer ON ballots(layer);
 
 CREATE TABLE layers (
     id INT PRIMARY KEY,
-    hare_output VARCHAR
+    hare_output VARCHAR,
+    hash CHAR(32),
+    aggregated_hash CHAR(32)
 ) WITHOUT ROWID;
 
 CREATE TABLE mesh_status (
     status SMALL INT PRIMARY KEY,
     layer INT NOT NULL
-) WITHOUT ROWID;
\ No newline at end of file
+) WITHOUT ROWID;
+
+CREATE TABLE rewards (
+    smesher CHAR(64),
+    coinbase CHAR(20),
+    layer INT NOT NULL,
+    total_reward UNSIGNED LONG INT,
+    layer_reward UNSIGNED LONG INT,
+    PRIMARY KEY (smesher, layer)
```

Comment:
i am weary of this primary key design. this constrains how we can write into database in the app. are we sure we don't ever want to write multiple rewards for the same smesher and layer? i think this should use ROWID instead

Reply:
smesher rewards are aggregated per layer. coinbase can't be changed in the same layer. i think this design is correct

Reply:
note that previously rewards were aggregated in the application, so there is definitely no reason to store multiple rows for the same smesher in the layer

Reply:
it's easy enough to change this before mainnet. but i think for db schema it's better not to assume how the app will save it.